webdriver_manager/utils.py

```diff
@@ -75,12 +76,17 @@ def os_type():
     return os_name() + str(os_architecture())
 
 
-def validate_response(resp):
+def validate_response(resp: requests.Response):
     if resp.status_code == 404:
         raise ValueError("There is no such driver by url {}".format(resp.url))
     elif resp.status_code != 200:
+        try:
+            resp_body = resp.json()
+        except JSONDecodeError:
+            resp_body = resp.content \
+                if getattr(resp, 'content', None) else str(resp)
         raise ValueError(
-            f'response body:\n{resp.json()}\n'
+            f'response body:\n{resp_body}\n'
```

Comment:
This all code could be replaced by `resp.text` in this row. @rachmadaniHaryono take a look please. We don't need convert to json here, because it will be also logged as a string in console without formatting.

Reply:
`f'response body:\n{resp.text}\n'`

i assume basically just that?

Reply:
`f'response body:\n{resp.text}\n'`

Reply:
@rachmadaniHaryono you can also test fallback locally that you did with your test to make sure this change works the same way your PR solution did before.

Reply:
i can but i have no response example and you already ok with different message, so i would not add anything anymore

current test is only to test raised error which is the core of the pr

maybe this change should be added to changelog when new version release